### PR TITLE
analytics: differentiate ci users

### PR DESCRIPTION
**Pull request by dberenbaum**
This would help to separate CI users in analytics. Depends on https://github.com/iterative/telemetry-python/pull/53.

Also needs a docs update in https://dvc.org/doc/user-guide/analytics.

### CODE REVIEW

**Comment by skshetry** [comment]
@dberenbaum, are you waiting for release? See https://github.com/iterative/dvc/wiki/Release-checklist#release-checklist. Otherwise let me know, I can cut a release as well.

**Comment by dberenbaum** [comment]
I guess I'm waiting for a release of https://github.com/iterative/telemetry-python before I can merge this.

**Comment by skshetry** [comment]
> I guess I'm waiting for a release of https://github.com/iterative/telemetry-python before I can merge this.

Right, that's what I meant. That checklist can be used to release any py-template based projects. Do you want to make a release yourself?

**Comment by skshetry** [comment]
Done. It’ll take a few minutes to get to pypi.

Btw, do you have permissions to make a release there? No need to make a draft tbh.

**Comment by dberenbaum** [comment]
> Btw, do you have permissions to make a release there? No need to make a draft tbh.

Yeah, but I thought it couldn't hurt this time to be a little cautious.

**Comment by dberenbaum** [comment]
Should be ready to merge now

File: dvc/analytics.py
Comment (skshetry) [inline dvc/analytics.py:110]: A private function may break in the future.

**Review by skshetry — APPROVED** [review]

**Comment by dberenbaum** [comment]
@skshetry Would you mind merging?

**Pull request merged**
